jean_zay/submitit/understanding-unets/exact_recon_unet_training.py

```diff
@@ -0,0 +1,47 @@
+from learning_wavelets.training_scripts.exact_recon_unet_training import train_unet
+from learning_wavelets.evaluation_scripts.exact_recon_unet_evaluate import evaluate_unet
+
+import numpy as np
+from scipy.stats import ttest_ind
+
+from jean_zay.submitit.general_submissions import train_eval_grid
+
+
+job_name = 'exact_recon_unet_training'
+n_epochs = 100
+batch_size = 8
+base_n_filters = 4
+n_layers = 4 
+non_linearity = 'relu'
+n_gpus = 1
+possible_std_dev = [0.0001, 5, 15, 20, 25, 30, 50, 55, 60, 75]
+
+base_parameters = dict(
+    n_epochs=n_epochs,
+    batch_size=batch_size,
+    base_n_filters=base_n_filters,
+    n_layers=n_layers,
+    non_linearity=non_linearity,
+)
+
+parameters = [
+    dict(**base_parameters)
```

Comment:
```suggestion
    base_parameters,
```

you don't need to create another dictionary here